### PR TITLE
[14.0][FIX] account_asset_operating_unit: fix reporting by OU

### PR DESCRIPTION
**Pull request by ps-tubtim**
This PR fixes reporting when generating asset reports with the asset group and operating unit.

### CODE REVIEW

File: account_asset_operating_unit/report/account_asset_report_xls.py [hunk 1]
```diff
@@ -28,20 +28,16 @@ def _get_asset_template(self):
     def _get_assets(self, wiz, data):
         super()._get_assets(wiz, data)
         if wiz.operating_unit_id:
-            # Filter asset by operating unit
             assets = data["assets"]
             assets_by_ou = assets.filtered(
                 lambda a: not a.operating_unit_id
                 or a.operating_unit_id == wiz.operating_unit_id
```
Comment (newtratip) [inline account_asset_operating_unit/report/account_asset_report_xls.py]: I think if filter by operating unit should show only assets that match with selected opearting unit on the wizard. Should not show assets that no have operating unit.

this line, if I change to
```
assets_by_ou = assets.filtered(lambda a: a.operating_unit_id == wiz.operating_unit_id)
```

Is this ok for you ? @ps-tubtim

Comment (ps-tubtim) [reply account_asset_operating_unit/report/account_asset_report_xls.py]: @newtratip I agree with you. I already change it.

**Review by Saran440 — APPROVED** [review]

**Review by Nantikan23 — APPROVED** [review]
Functional Tested

**Pull request merged**